internal/provider/client.go

```diff
@@ -90,6 +90,12 @@ type ApiClient interface {
 	UpdateLogsPipeline(ctx context.Context, req *models.CreateOrUpdateLogsPipelineConfigRequest) (*models.LogsPipelineConfig, error)
 	DeleteLogsPipeline(ctx context.Context) error
 
+	// Metrics Aggregator
+	CreateMetricsAggregator(ctx context.Context, req *models.CreateOrUpdateMetricsAggregatorConfigRequest) (*models.MetricsAggregatorConfig, error)
```

Comment:
Aggregation

Reply:
it was my bad that the models are `GetMetricsAggregatorConfig` and not `GetMetricsAggregationConfig` but we should keep it as much as possible aggregation
like the path (`/api/aggregations/metrics/`)